lib/kitchen/provisioner/install.erb

```diff
@@ -72,8 +72,72 @@ else
   exit 2
 fi
 
-#{install_chef}
+echo "-----> Installing other dependencies..."
+YUM_CMD="$(which yum)"
+APT_CMD="$(which apt-get)"
+APK_CMD="$(which apk)"
+if [[ -z "$(which git)" ]]; then
```

Comment:
It would be nice to have the scripts as separate `.sh` files. We can lint them to make sure they are `sh` compatible and will work well in different environments.